clifford/numba/_multivector.py

```diff
@@ -64,7 +67,7 @@ def _numba_type_(self):
     except KeyError:
         # Computing and hashing `dtype_type` is slow, so we do not use it as a
         # hash key. The raw numpy dtype is much faster to use as a key.
-        dtype_type = _numpy_support.from_dtype(dt)
+        dtype_type = _typeof_ndarray(self.value, None)
         ret = cache[dt] = MultiVectorType(layout_type, dtype_type)
```

Comment:
This cache is now unsafe - you're caching both `int, C` and `int, F` under the same key, `int`.

Reply:
I guess we could cache with the whole `dtype_type` or maybe a tuple `(int, C)`?

Reply:
Caching with the whole `dtype_type` might be too slow, see the comment above.